lib/JSON/CalDAV/CalendarHandler.php

```
@@ -0,0 +1,476 @@
+
+                // see vendor/sabre/dav/lib/CalDAV/SharingPlugin.php:268
+                $this->server->httpResponse->setHeader('X-Sabre-Status', 'everything-went-well');
+
+                return [200, null];
+            }
+        }
+
+        return [400, null];
+    }
+
+    public function changePublicRights($request, $response) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$response'. (undefined)

(UnusedFormalParameter)